README.md

```diff
@@ -57,7 +57,7 @@ if __name__ == "__main__":
 ##### Configure the default answer
 Add to `slackbot_settings.py` a default_reply:
 ```python
-default_reply = "Sorry but I didn't understood you" 
+default_reply = "Sorry but I didn't understood you"
 ```
```

Comment:
trivial thing: add the space back in this file so it disappears from the diff.  our goal is to have the minimal amount of changes to enhance readability in reviews like this one.